apps/challenges/aws_utils.py

```diff
@@ -616,6 +616,112 @@ def scale_workers(queryset, num_of_tasks):
     return {"count": count, "failures": failures}
 
 
+def scale_resources(challenge, worker_cpu_cores, worker_memory):
+    """
+    The function called by scale_resources_by_challenge_pk to send the AWS ECS request to change the resources used by
+    a challenge's workers.
+
+    Deregisters the old task definition and creates a new definition that is substituted into the challenge workers.
+
+    Parameters:
+    challenge (): The challenge object for whom the task definition is being registered.
+    new_cpu_val (int): The CPU value (1 vCPU = 1024 values) that should be assigned to workers.
```

Comment:
rename to `worker_cpu_cores`  and change `The CPU value (1 vCPU = 1024 values)` to `vCPU (1 CPU core = 1024 vCPU)`